tuftool/CHANGELOG.md

```diff
@@ -93,7 +99,8 @@ Major update: much of the logic in `tuftool` has been factored out and added to
 ### Added
 - Everything!
 
-[Unreleased]: https://github.com/awslabs/tough/compare/tuftool-v0.5.0...develop
+[Unreleased]: https://github.com/awslabs/tough/compare/tuftool-v0.6.0...develop
```

Comment:
Why do we add [Unreleased] in `tuftool` but not for `tough-kms` and `tough-ssm` ? Is it because the may not get any new changes ?

Reply:
I don't think there's a well-reasoned purpose for the discrepancy. This line is invisible in the markdown rendering, but is convenient if we choose to add a `[Unreleased]` section back to the top for collecting unreleased changes over time. I think your observation that we are less likely to make such a section in `tough-kms` or `tough-ssm` is as good an explanation as any.